### ISSUE TITLE
Chinese texture mistakes

### ISSUE THREAD

**Issue by GhostlyDark** [GhostlyDark/OoT-Reloaded#6]
> ![image](https://user-images.githubusercontent.com/45894930/147278320-5c285760-df8e-4c52-b081-b8198f837edb.png)
> ![image](https://user-images.githubusercontent.com/45894930/147278352-e7b88721-c258-43eb-8057-6ab53c22c715.png)
> I tried the OOT texture pack, you wrote 攻击 as 玫击, which is wrong. Hope you can change that texture to 攻击。
> ![image](https://user-images.githubusercontent.com/45894930/147278691-1b009e60-2d0a-46ac-bd76-5a99b734874c.png)
> ![image](https://user-images.githubusercontent.com/45894930/147278723-52e983f4-332e-45c1-b7cb-23b560f5cfb0.png)
> 迷失的塞林 is wrong. It should be 迷失的森林
> 
> _Originally posted by @Darth-Koopa in https://github.com/gonetz/GLideN64/issues/2626#issuecomment-1000462406_

@Kaoffie typed out most of the text for me. For the Action Prompts (blue button text), I was using an OCR. Looks like some mistakes slipped through, so thanks for reporting that @Darth-Koopa.

If anyone wants to double check things, here's the text base. No text in the Chinese Traditional column means that OoT ChT used the same text as OoT ChS.
[ChS_ChT.docx](https://github.com/GhostlyDark/OoT-Reloaded/files/7774217/ChS_ChT.docx)

**Comment by Darth-Koopa** [comment]
![image](https://user-images.githubusercontent.com/45894930/147356016-b3d0b61f-e330-41fc-97ed-b278ca0d27bb.png)
change to 迷失的森林
![image](https://user-images.githubusercontent.com/45894930/147355592-0ef84b13-584c-4f34-af57-1222c524bb86.png)
黯之勋章is wrong. It should be 暗之勋章
![image](https://user-images.githubusercontent.com/45894930/147355670-2c02e2a3-9610-40be-9a10-ae7cf1d85e4c.png)
光之前揍曲 is wrong. It should be 光之前奏曲
![image](https://user-images.githubusercontent.com/45894930/147355849-197839d3-5b1b-4766-b232-9fd9e7c13ac4.png)
冰剑（魔法点2）is wrong. It should be 冰箭（魔法点2）
![image](https://user-images.githubusercontent.com/45894930/147355882-91e5083c-9161-4ba5-bafe-748b22cf6581.png)
光箭（魔法点4）should be 光箭（魔法点2）

**Comment by GhostlyDark** [comment]
The change from 2 to 4 MP for the light arrow text is intentional. That's a mistake that the ChS ROM contains. It uses 4 magic points as any other official translation suggests, including ChT.

toDo:
```
玫击		->	攻击		Attack
迷失的塞林	->	迷失的森林	Lost Woods
黯之勋章 	->	暗之勋章		Shadow Medallion
光之前揍曲	->	光之前奏曲	Prelude of Light
冰剑（魔法点2）	->	冰箭（魔法点2）	Ice Arrow (2MP)
```

**Comment by GhostlyDark** [comment]
Fixed in `v10.1.0`
